### DISCUSSION TITLE
Negative aspects of Maui

### DISCUSSION THREAD

**Discussion by sacdevops** [dotnet/maui#3706] [General]
I only ever hear the positive aspects about .NET Maui. I would be interested in what you personally don't like or just missing in Maui and would see in near future.
In order not to start great discussions (which you can move into other discussions), this should simply be a collection point so that the Maui team can also access it quickly. So just give it a short and concise list and no whole novels, please. 😁

Would be good quality research to make the product better in the near future.

I'm exited for your opinions. 😁

**Comment by saint4eva** [comment]
I do not like the xaml and mvvm aspect of it

**Reply by GeraudFabien** [reply]
Missing Linux and web support (Some may add W7 too i personally don't care but if you work with Chinese people you may)
Visual studio 2022 needed (Maui check isn't officially supported)
Pixel perfect in early stage still experimental and not officially support.
Not release still in preview.

Again no web support !!!! It mean you can't target web and you can't target electron.

**Reply by saint4eva** [reply]
Every aspect of Maui is still emperimental.

**Reply by codeaphex** [reply]
Web support is a must imo.
XAML and MVVM have to stay though, they are awesome, the dialects could be merged sometime in the future (WPF/WinUI/MAUI)

**Reply by saint4eva** [reply]
Xaml dialect of Xamarin is specifically for mobile scenario. So, trying to unify all the xaml dialects would be impossible to do - as there would be a lot of compromises in order to find common denominators. 

In as much as web support is important, having a good support for MVU C# UI is inevitably important - a must have feature and support.
